### PR TITLE
transpiler integration doc

### PR DESCRIPTION
**Pull request by tailsu**

### CODE REVIEW

**Comment by atanasovg** [comment]
ping @ikoevska for a review and approval

File: transpilers.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+nav-title: Using transpilers in NativeScript
+title: "App: Transpilers"
+description: Learn how to enable TypeScript and other transpiler support in your project.
+position: 5
+---
+
+# Using transpilers in NativeScript
+
+Transpilers allow you to develop your application in languages other than JavaScript, CSS and XML. NativeScript applications can be developed more comfortably when transpilers are used in the project. NativeScript projects support two transpilers out-of-the-box - TypeScript and Babel, although any other transpiler can be integrated through the necessary hooks.
```
Comment (ErjanGavalji) [inline transpilers.md:10]: I'd use "respective hooks" instead of "necessary hooks"...

**Comment by ErjanGavalji** [comment]
:+1:

**Comment by teobugslayer** [comment]
I've talked to @ikoevska in person and we decided to include this PR as is for 1.5. After that; we shall revisit this article and fix the issues we find.

**Pull request merged**